app/backend/utils.js

```diff
@@ -1,5 +1,3 @@
 module.exports.checkPasswordLength = function (password, response){
-  if(password.length < 6){
-    response.status(400).send({ errmsg: 'Password length must be at least 6.' })
-  }
+  return (password.length < 6)
```

Comment:
Password validation might be done in only one place for better management of the flow. As far as I remember, we as well check the password against the common ones and we also check if the password is supplied in the first place when the user is created. All those validations might be in one place, inside User model definition, to keep things under control.